### PR TITLE
eaf-pdf-viewer: run scroll hooks

### PR DESCRIPTION
**Pull request by c1-g**
Have each of Python's scroll functions run `eaf-pdf-scroll-up-hook`/`eaf-pdf-scroll-down-hook`/`eaf-pdf-scroll-left-hook`/`eaf-pdf-scroll-right-hook` depending on what it does.

### CODE REVIEW

File: app/interleave/eaf-interleave.el [hunk 1]
```diff
@@ -89,6 +90,11 @@ taken as columns."
   :type 'boolean
   :group 'eaf)
 
+(defcustom eaf-interleave--find-note-path-function 'eaf-interleave--find-note-path-default
+  "TODO"
```
Comment (MatthewZMD) [inline app/interleave/eaf-interleave.el:94]: Did you forget something here?

**Comment by MatthewZMD** [comment]
@luhuaei Could you help reviewing this PR?

File: app/interleave/eaf-interleave.el [hunk 2]
```diff
@@ -141,47 +147,72 @@ split horizontally."
...
+      (eaf-interleave-open-notes-file))
+    (setq org-window (get-buffer-window eaf-interleave-org-buffer))
+    (with-current-buffer eaf-interleave-org-buffer
+      (when-let* ((page-list (mapcar #'string-to-number
+                                     (org-property-values eaf-interleave--page-note-prop)))
+                  (note-on-this-page (cdar (sort
+                                            (mapcar #'(lambda (value)
+                                                        (cons (abs (- page-pdf value)) value))
+                                                    page-list)
+                                            #'(lambda (cons another-cons)
```
Comment (MatthewZMD) [inline app/interleave/eaf-interleave.el:171]: iirc `#'` before `(lambda ` is unnecessary.

**Comment by c1-g** [comment]
I'm sorry, this is supposed to be a small commit those things in eaf-interleave is unrelated.

**Pull request closed without merging**